grpc/server/src/chronicles.rs

```diff
@@ -351,7 +403,7 @@ fn read_chronicle_template(
     let start = FAtom::from(start);
 
     let end: FAtom = match action_kind {
-        ChronicleKind::Problem => panic!("unsupported case"),
+        ChronicleKind::Problem => return Err(anyhow!("Problem type not supported")),
```

Comment:
The `bail!("Problem...")` macro from anyhow would do the same thing.

Reply:
Noted.